eng/targets/Packaging.targets

```diff
@@ -1,9 +1,10 @@
 <Project>
 
+  <!-- Last released version was 3.1.32 - make sure the baseline stays there -->
```

Comment:
CC @dougbu on this change

Reply:
This is fine but should be unnecessary because (a) we'll never change `$(AspNetCoreBaselineVersion)` again and (b) we'll probably leave `$(ValidateBaseline)` set to `false` if we ever need to build internally.

Nothing to see here 😁

Reply:
It was failing before because `PreviousAspNetCoreReleaseVersion` = current version minus 1. Right now both are 3.1.32, so the comparison failed. It would continue failing going forward because current version will keep incrementing. `ValidateBaseline=false` would work too.